deploy/helm/kubecf/templates/brain-tests.yaml

```diff
@@ -0,0 +1,223 @@
+# psp-role       | predefined ? -> scf default account
+# configgin-role | predefined ? -> scf default account
```

Comment:
I don't understand these notes.

Reply:
Yes, I should remove that. When I wrote this I was unsure if kubecf had default roles I could bind too. Answer: no. Which is why I then merged the relevant permissions into the test role.

Reply:
Done with commit b4a6f34cf652a6e983167b8319a88217acada94f